utils/benchmark/ebizzy-0.3/ebizzy.c

```diff
@@ -228,9 +228,10 @@ static void read_options(int argc, char *argv[])
 
 static void touch_mem(char *dest, size_t size)
 {
+	long s = (long)size;
 	int i;
 	if (touch_pages) {
-		for (i = 0; i < size; i += page_size)
+		for (i = 0; i < s; i += page_size)
```

Comment:
I'd just use casting where possible, e.g.
`for (i = 0; i < (long)size; i += page_size)`